bot/exts/moderation/modpings.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, bot: Bot):
         )
 
     async def fetch_guild(self) -> None:
-        """Fetch the guild."""
+        """Fetch the configured constants.Guild.id guild object and assign to self.guild."""
```

Comment:
Really picky but variables should go in backquotes to indicate that it's a variable (this would apply to both `constants.Guild.id` and `self.guild`.